search_indices/backends/faceted_attribute_backend.py

```diff
@@ -28,13 +28,23 @@ class FacetedAttributeBackend(BaseFilterBackend, FilterBackendMixin):
     faceted_search_param = "facet_attribute"
 
     @staticmethod
-    def get_attributes(documents: List[Type[Document]] = None) -> list:
+    def get_attributes(
+        documents: List[Type[Document]] = None, authenticated=True
```

Comment:
`List` and `Type` have been deprecated since 3.9:
https://docs.python.org/3.9/library/typing.html#typing.Type
https://docs.python.org/3.9/library/typing.html#typing.List
```suggestion
        documents: list[type[Document]] = None, authenticated=True
```